examples/simple/go.mod

```
@@ -0,0 +1,11 @@
+module github.com/newrelic/opentelemetry-exporter-go/examples
```

Should this be more like
```suggestion
module github.com/newrelic/opentelemetry-exporter-go/examples/simple
```
?

---

Yes, thanks, I'll be putting in a gin example beside it soon.